config/config_test.go

```diff
@@ -70,6 +71,7 @@ txn-total-size-limit=2000
 [tikv-client]
 commit-timeout="41s"
 max-batch-size=128
+region-cache-ttl=6000
```

Comment:
It seems that we don't check it. `region-cache-ttl` is written to configure file, but we don't check it.